### DISCUSSION TITLE
favicon

### DISCUSSION THREAD

**Discussion by pacmandoh** [vuepress-theme-hope/vuepress-theme-hope#3212] [Q&A] [answered]
尝试多种配置，始终无法在移动端ipadOS, IOS上edge/chrome显示favicon(safari正常显示)，但电脑端safari和edge都能正常显示。
后尝试直接pnpm新建项目，发现移动端还是无法显示favicon。
- devDependencies:
- @vuepress/client 2.0.0-beta.62
- vue 3.3.4
- vuepress 2.0.0-beta.62
- vuepress-plugin-search-pro 2.0.0-beta.220
- vuepress-theme-hope 2.0.0-beta.220

**Comment by Mister-Hope** [comment, marked as answer]
proove what you are saying with latest  version
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/8673c8b1-87dc-4424-a7f3-282cfc04d1c5)

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/947b4478-6142-44da-b81a-bffb914a9734)

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/1b7396fb-5aa6-4a3c-99a7-aa44aa4389ff)

**Reply by Mister-Hope** [reply]
My blog is also working fine https://mrhope.site

Try not to waste others' time (5min for me), you are marked and your further reports will all need valid reproduction.

**Reply by pacmandoh** [reply]
实在抱歉！浪费您的时间
